organization/presentations.md

```diff
@@ -14,12 +14,15 @@ layout: default
 
 ## 2018
 
+* [HEP Software Foundation (R&D) activities](https://indico.cern.ch/event/765782/contributions/3178923/attachments/1739367/2814552/EduardoRodrigues_HSF-2018-10-23.pdf), ECOM2X WG7 Meeting, CERN, 23 October 2018, Eduardo Rodrigues
```

Comment:
This appears to be a protected link

Reply:
Ah, I forgot to check that, and was signed-in. Fixed now, thanks.